server/services/lists/comCaseloadService.ts

```diff
@@ -348,4 +376,102 @@ export default class ComCaseloadService {
       'licence status is not one of ACTIVE, VARIATION_IN_PROGRESS, VARIATION_SUBMITTED, VARIATION_APPROVED, VARIATION_REJECTED, REVIEW_NEEDED'
     )
   }
+
+  createCaseloadViewModel = (caseload: ManagedCase[]): ComCase[] => {
+    return caseload
+      .map(c => {
+        const licence = this.findLicenceToDisplay(c)
+        const releaseDate = c.nomisRecord.releaseDate || c.nomisRecord.conditionalReleaseDate
```

Comment:
**suggestion:** I think here we should be using the licence release date if it exists, then falling back to NOMIS if it doesn't (eg if the licence is `NOT_STARTED`

Reply:
done, updated to use licence.releaseDate if available